pkg/scheduler/partition.go

```diff
@@ -1070,44 +1070,32 @@ func (pc *PartitionContext) GetRejectedApplications() []*objects.Application {
 	return appList
 }
 
-// getAppsByState returns a slice of applicationIDs for the current applications filtered by state
-// Completed and Rejected applications are tracked in a separate map and will never be included.
-func (pc *PartitionContext) getAppsByState(state string) []string {
+func (pc *PartitionContext) getAppsByInternalState(appMap map[string]*objects.Application, state string) []string {
```

Comment:
the name `getAppsState`  should be good here since there is no such internal state.

Reply:
Sure, updated the function name to ```getAppsState```.